api-playground/openapi/setup.mdx

```diff
@@ -61,30 +50,30 @@ You can add an `openapi` field to an object in the `tabs` or `anchors` array in
 
 ### Create MDX files for OpenAPI endpoints
 
-You can also create MDX files associated with each OpenAPI endpoint if you would like to add additional context to specific pages. Check out an example from [Elevenlabs](https://elevenlabs.io/docs/api-reference/speech-to-speech) below. 
+You can also create MDX files associated with each OpenAPI endpoint if you would like to add additional context to specific pages or have more granular control over the navigation. Here is [the code](https://github.com/elevenlabs/elevenlabs-docs/blob/main/api-reference/speech-to-speech.mdx?plain=1#L2) from [Elevenlabs](https://elevenlabs.io/docs/api-reference/speech-to-speech).
 
 ![](/images/elevenlabs-mdx-autogeneration-example.png)
 
-You can autogenerate these MDX files from your OpenAPI document using our scraper, or create each page manually.
+If you want to autogenerate MDX files for every endpoint in your OpenAPI document you can use our scraper.
 
-#### Autogenerate files (Recommended)
+#### Autogenerate files
 
-Our Mintlify [scraper](/api-playground/openapi/setup#autogenerate-files-recommended) helps autogenerate MDX files for your OpenAPI endpoints. Use the relative path to the OpenAPI document in your codebase. If you're using a publicly-hosted OpenAPI document, you can just supply the URL in place of a path.
+Our Mintlify [scraper](/api-playground/openapi/setup#autogenerate-files-recommended) autogenerates MDX files for your OpenAPI endpoints. Use the relative path to the OpenAPI document in your codebase. If you're using a publicly-hosted OpenAPI document, you can supply the URL in place of a path.
```

Comment:
looks like a broken link? it was self-referential before, but maybe it should link to the actual scraper package?
```suggestion
Our Mintlify [scraper](https://www.npmjs.com/package/@mintlify/scraping) autogenerates MDX files for your OpenAPI endpoints. Use the relative path to the OpenAPI document in your codebase. If you're using a publicly-hosted OpenAPI document, you can supply the URL in place of a path.
```